### PR TITLE
Version Bump

### PR DESCRIPTION
**Pull request by LionOps**

### CODE REVIEW

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # [Changelog](https://github.com/yola/pixabayjs/releases)
 
+## 0.5.0
+* `pixabayjs` API changes ([#19][19])
+    - `pixabayjs.defaults` is now an object instead of a function
+    - `pixabayjs.requestFactory` has been replaced with `pixabayjs.resultList`, which returns a new `ResultList`. The `RequestFactory` no longer exists and thus there is no intermediate object.
```
Comment (euwest) [inline CHANGELOG.md]: this can just be:

> - intermediary `RequestFactory` removed

**Comment by snitch** [comment]
:sparkles: No lint errors found. :sparkles:

**Comment by euwest** [comment]
:+1:

**Pull request merged**